src/main/java/org/eclipse/keyple/plugin/stub/StubPoolPluginFactoryBuilder.java

```diff
@@ -0,0 +1,100 @@
+/* **************************************************************************************
+ * Copyright (c) 2021 Calypso Networks Association https://calypsonet.org/
+ *
+ * See the NOTICE file(s) distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License 2.0 which is available at http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ************************************************************************************** */
+package org.eclipse.keyple.plugin.stub;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.eclipse.keyple.core.util.Assert;
+import org.eclipse.keyple.plugin.stub.StubPoolPluginFactoryAdapter.StubPoolReaderConfiguration;
+
+/**
+ * Builds instances of {@link StubPoolPluginFactory}.
+ *
+ * @since 2.0
+ */
+public final class StubPoolPluginFactoryBuilder {
+
+  /**
+   * The plugin default name
+   *
+   * @since 2.0
+   */
+  public static String PLUGIN_NAME = "StubPoolPlugin";
+
+  private StubPoolPluginFactoryBuilder() {}
+
+  /**
+   * Creates builder to build a {@link StubPoolPluginFactory}.
+   *
+   * @return created builder
+   * @since 2.0
+   */
+  public static StubPoolPluginFactoryBuilder.Builder builder() {
+    return new StubPoolPluginFactoryBuilder.Builder();
+  }
+
+  /**
+   * Builder to build a {@link StubPoolPluginFactory}.
+   *
+   * @since 2.0
+   */
+  public static class Builder {
```

Comment:
private static

Reply:
how can users access it, if it's private?